library/std/src/lib.rs

```diff
@@ -143,8 +143,14 @@ macro_rules! unreachable {
     ($($msg:literal)? $(,)?) => (
         kani::panic(concat!("internal error: entered unreachable code: ", $($msg)?))
     );
+    // Needed for 2018 and older rust editions.
+    // TODO: Is it possible to trigger an error if 2021 and above?
+    ($($msg:expr)? $(,)?) => (
```

Comment:
Can you add a comment on each rule that mentions example of an `unreachable` statement captured by this rule (it is not immediately clear to me what this new rule captures)?

Reply:
Sure.